site/ua.njk

```diff
@@ -0,0 +1,152 @@
+---
+layout: default
+meta_title: Рух, покликаний зупинити Пандемію COVID-19
+meta_description: Наші уряди не можуть зробити це самотужки, ми повинні вжити заходів вже зараз! Зробіть свій внесок і залишайтеся блядь вдома!
+body_class: home
+title: '#StayTheFuckHome'
+lang: ua
+---
+
+<main id="main" class="main inner flex flex-1 flex-col py-10 lg:py-20 focus:outline-none" tabindex="-1">
+
+    <section id="container-centre" class="column centre flex-1">
+
+        <h1 class="page-title">{{ title }}</h1>
+        <h2 class="page-subtitle">Рух, покликаний зупинити Пандемію COVID-19</h2>
+
+        <div class="content">
+
+            <p class="text-xl">
+                Нашим урядам не вдається запобігти поширенню
+                <a href="https://uk.wikipedia.org/wiki/SARS-CoV-2">SARS-CoV-2</a>
+                і стримати пандемію <a href="https://uk.wikipedia.org/wiki/%D0%9A%D0%BE%D1%80%D0%BE%D0%BD%D0%B0%D0%B2%D1%96%D1%80%D1%83%D1%81%D0%BD%D0%B0_%D1%85%D0%B2%D0%BE%D1%80%D0%BE%D0%B1%D0%B0_2019">Коронавірусної інфекції COVID-19</a>.
+                Повільна реакція, політика умиротворення громадськості і прагнення стабілізувати економіку утримують їх
+                від вжиття заходів, необхідних для захисту мільйонів людей від цього захворювання. Прийшов час для нас, громадян цієї землі,
+                вжити заходів зараз і внести свій вклад в боротьбу з COVID-19.
+                <br /><br />
+                Коротше кажучи: <strong>Сидіть, блядь, вдома!</strong>
+            </p>
+
+            <h2 class="text-3xl">Маніфест Само-Карантину</h2>
+
+            <p>
+                При відсутності добре вивченого лікування і працюючої вакцини, яка буде випущена як мінімум ще через рік,
```

Comment:
За відсутності

Reply:
"За відсутності" applies only to people ("whom" rather than "what") in ukrainian language. 
That is why in this case it is still correct "При відсутності"

Reply:
Так. За відсутності кого, при відсутності чого. Все вірно.